coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -106,6 +110,18 @@ def markers(self):
         """
         return self._markers
 
+    @property
+    def metadata(self):
+        """
+        A namedtuple of documentation symbols.
+
+        These documentation symbols are used to define certain symbols that
+        define a documentation comment.
+
+        Currently consists of ``param_start``, ``param_end`` and ``return_sep``.
```

Comment:
if someone is interested, he/she will look into the `Metadata` namedtuple and see all members. Especially if you change `Metadata` you also have to maintain this little sentence :D